### PR TITLE
Loong horizon task cookbook

### PR DESCRIPTION
**Pull request by derrickchoi-openai**
## Summary

Added a new cookbook on running long horizon tasks in Codex

## Motivation

A lot of customers are asking about how to run long running tasks with Codex

---

## For new content

When contributing new content, read through our [contribution guidelines](https://github.com/openai/openai-cookbook/blob/main/CONTRIBUTING.md), and mark the following action items as completed:

- [X] I have added a new entry in [registry.yaml](https://github.com/openai/openai-cookbook/blob/main/registry.yaml) (and, optionally, in [authors.yaml](https://github.com/openai/openai-cookbook/blob/main/authors.yaml)) so that my content renders on the cookbook website.
- [X] I have conducted a self-review of my content based on the [contribution guidelines](https://github.com/openai/openai-cookbook/blob/main/CONTRIBUTING.md#rubric):
  - [X] Relevance: This content is related to building with OpenAI technologies and is useful to others.
  - [X] Uniqueness: I have searched for related examples in the OpenAI Cookbook, and verified that my content offers new insights or unique information compared to existing documentation.
  - [X] Spelling and Grammar: I have checked for spelling or grammatical mistakes.
  - [X] Clarity: I have done a final read-through and verified that my submission is well-organized and easy to understand.
  - [X] Correctness: The information I include is correct and all of my code executes successfully.
  - [X] Completeness: I have explained everything fully, including all necessary references and citations.

We will rate each of these areas on a scale from 1 to 4, and will only accept contributions that score 3 or higher on all areas. Refer to our [contribution guidelines](https://github.com/openai/openai-cookbook/blob/main/CONTRIBUTING.md) for more details.

### CODE REVIEW

File: examples/codex/long_horizon_tasks.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+- [Automations](https://developers.openai.com/codex/app/automations/) (routine work in the background)
+- [Git worktrees](https://developers.openai.com/codex/app/worktrees/) (isolate runs, keep diffs reviewable, reduce thrash)
+
+![Codex app workspace with project thread](../../images/long_horizon_codex_app_main_workspace.jpg)
+
+## My setup for the test
+
+I picked a design tool for this “experiment” because it’s an unforgiving test: UI + data model + editing operations + lots of edge cases. You can’t bluff it. If the architecture is wrong, it breaks quickly.
+
+I gave GPT-5.3-Codex a meaty spec, ran it at “Extra High” reasoning, and it ended up running uninterrupted for ~25 hours and was able to stay coherent and ship quality code. The model also ran verification steps (tests, lint, typecheck) for every milestone it completed.
```
Comment (charlie-openai) [inline examples/codex/long_horizon_tasks.md:77]: We might want to say "I used plan mode" now that that's available. Generally trying to have devs not write their own specs and instead iterate with plan mode.

**Review by charlie-openai — APPROVED** [review]
